src/rust/engine/workunit_store/src/lib.rs

```diff
@@ -99,17 +115,33 @@ pub struct WorkUnitStore {
 
 #[derive(Default)]
 pub struct WorkUnitInnerStore {
+  graph: WorkunitGraph,
+  span_id_to_graph: HashMap<SpanId, NodeIndex<u32>>,
   workunit_records: HashMap<SpanId, WorkunitRecord>,
   started_ids: Vec<SpanId>,
   completed_ids: Vec<SpanId>,
   last_seen_started_idx: usize,
   last_seen_completed_idx: usize,
 }
 
+fn should_display_workunit(workunit_records: &HashMap<SpanId, WorkunitRecord>, id: &str) -> bool {
+  let record = match workunit_records.get(id) {
+    None => return false,
+    Some(record) => record,
+  };
+  let metadata = match record {
+    WorkunitRecord::Started(StartedWorkUnit { metadata, .. }) => metadata,
+    WorkunitRecord::Completed(WorkUnit { metadata, .. }) => metadata,
```

Comment:
So, it doesn't feel like we should ever display a Completed workunit in heavy_hitters? It would only render "running" (ie, Started) workunits.

Reply:
good point.